django/db/backends/base/introspection.py

```diff
@@ -113,29 +113,34 @@ def sequence_list(self):
         all apps.
         """
         from django.apps import apps
-        from django.db import models, router
+        from django.db import router
 
         sequence_list = []
+        cursor = self.connection.cursor()
 
         for app_config in apps.get_app_configs():
             for model in router.get_migratable_models(app_config, self.connection.alias):
                 if not model._meta.managed:
                     continue
                 if model._meta.swapped:
                     continue
-                for f in model._meta.local_fields:
-                    if isinstance(f, models.AutoField):
-                        sequence_list.append({'table': model._meta.db_table, 'column': f.column})
-                        break  # Only one AutoField is allowed per model, so don't bother continuing.
-
+                sequence_list.extend(self.get_sequences(cursor, model._meta.db_table))
                 for f in model._meta.local_many_to_many:
                     # If this is an m2m using an intermediate table,
                     # we don't need to reset the sequence.
                     if f.remote_field.through is None:
-                        sequence_list.append({'table': f.m2m_db_table(), 'column': None})
+                        sequence_list.extend(self.get_sequences(cursor, f.m2m_db_table()))
 
         return sequence_list
 
+    def get_sequences(self, cursor, table_name):
+        """
+        Return a list of introspected sequences for table_name. Each sequence
+        is a dict: {'table': <table_name>, 'column': <column_name>}. An optional
+        'name' key can be added if the backend supports named sequences.
+        """
+        return []
```

Comment:
Would it be better to raise `NotImplementedError()` so backends don't overlook implementing it? They can always return an empty list if it's not supported.